Source/Csla/DataPortalClient/HttpProxyOptions.cs

```diff
@@ -36,6 +36,16 @@ public class HttpProxyOptions
     /// </summary>
     public Func<IServiceProvider, HttpClient> HttpClientFactory { get; set; } = DefaultHttpFactory;
 
+    /// <summary>
+    /// Gets or sets the timeout value for the HTTP request (only sync calls).
+    /// </summary>
+    public TimeSpan Timeout { get; set; }
```

Comment:
This timeout is used for sync and async calls. So please remove the ("only sync calls") here.